site/src/modules/provisioners/ProvisionerStatusAlert.tsx

```diff
@@ -1,17 +1,19 @@
 import type { AlertColor } from "components/Alert/Alert";
 import type { FC } from "react";
-import { ProvisionerAlert } from "./ProvisionerAlert";
+import { AlertVariant, ProvisionerAlert } from "./ProvisionerAlert";
 
 interface ProvisionerStatusAlertProps {
 	matchingProvisioners: number | undefined;
 	availableProvisioners: number | undefined;
 	tags: Record<string, string>;
+	variant?: AlertVariant;
```

Comment:
Should we mark `variant` non optional?

Reply:
The component uses a reasonable default for the variant. So I think optional is fine. Is there a specific concern that could be alleviated by making it non-optional?